medicines/api/src/product.rs

```diff
@@ -1,28 +1,18 @@
 use crate::{
-    document::{self, get_documents_graph_from_documents_vector, Document},
+    document::{self, get_documents},
     substance::Substance,
 };
+use juniper::FieldResult;
 use search_client::{models::IndexResult, Search};
 
 #[derive(Debug, Eq, Ord, PartialEq, PartialOrd)]
 pub struct Product {
     name: String,
-    document_count: i32,
```

Comment:
Should keep this around so we don't have to make round trips to Azure for every product just to get the document count.